app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  has_many :comments, class_name: 'Comment', foreign_key: 'author_id'
+  has_many :posts, class_name: 'Post', foreign_key: 'author_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'author_id'
+
+  def recent_posts(limit = 3)
+    posts.order(created_at: :desc).limit(limit)
+  end
+
+  def update_posts_counter
+    update(post_counter: posts.count)
+  end
```

Comment:
- [ ] **[OPTIONAL]** You already have a method to update `post_counter` on your `Post` model. It's safe to remove this one! 🤓 🔧